static/js/portico/landing-page.js

```diff
@@ -134,13 +136,16 @@ var apps_events = function () {
 
     $(".apps a .icon").click(function (e) {
         var next_version = $(e.target).closest('a')
-            .attr('href')
+            .attr('class')
             .replace('/apps/', '');
         version = next_version;
 
         update_path();
         update_page();
-        $("body").animate({ scrollTop: 0 }, 200);
+        // Jquery code for scrolling on top with animation when click on apps icon
+        $('html, body').animate({
+            scrollTop: $("#apps_flex_id").offset().top,
```

Comment:
agree with you, fixing

Reply:
@alexandraciobica  oh forget to tell you problem, adding html or body is a problem for different size screen hence it does not redirect to download button correctly, for some cases it goes to button only from the end and in some cases it goes to top not showing button, that why i add **id**

Reply:
Can you show me what are those cases? Because I tried it out and only for 320x480 screen size it was a problem and the button was not entirely in the screen.

If this is the only case, I would rather choose to take the scroll to the top for all the users because phones with 3/2 screen ratio are not produced anymore and from Zulip's users probably there are at most 1% users that probably still have this kind of screen, if there are at all. I think it is a better user experience to see the top bar, and I would make this compromise of showing the top bar for most probably 99% of users (maybe more) and not showing the entire button for the ones with 320x480. 
What do you think @timabbott?

Reply:
did you check on responsive design mode or tried on real screen, i guess my observation could be wrong as i checked on firefox responsive design mode and it mostly give wrong simulation of screen

Reply:
i think we should add body and remove id, I think you have done more research than me:sweat_smile: